remotespark/utils/constants.py

```diff
@@ -68,3 +68,7 @@
 DELETE_SESSION_ACTION = "delete"
 START_SESSION_ACTION = "start"
 DO_NOTHING_ACTION = "nothing"
+
+INTERNAL_ERROR_MSG = "An internal error was encountered.\n" \
+                     "Please file an issue at https://github.com/jupyter-incubator/sparkmagic\nError:\n{}"
```

Comment:
We write it to the logger rather than exposing it directly to the user: https://github.com/jupyter-incubator/sparkmagic/pull/216/files/ab618e5e802659f92ac55af90614279a61e055ea#diff-dc25b4292d5e26ef9ba1605e37770b67R64